src/Umbraco.Web/Runtime/WebRuntimeComposer.cs

```diff
@@ -238,7 +239,17 @@ public override void Compose(Composition composition)
 
             // register core CMS dashboards and 3rd party types - will be ordered by weight attribute & merged with package.manifest dashboards
             composition.WithCollectionBuilder<DashboardCollectionBuilder>()
-                .Add(composition.TypeLoader.GetTypes<IDashboard>());
+                .Add<ContentDashboard>()
+                .Add<ExamineDashboard>()
+                .Add<FormsDashboard>()
+                .Add<HealthCheckDashboard>()
+                .Add<ManifestDashboard>()
+                .Add<MediaDashboard>()
+                .Add<MembersDashboard>()
+                .Add<PublishedStatusDashboard>()
+                .Add<RedirectUrlDashboard>()
+                .Add<RedirectUrlDashboard>()
+                .Add<SettingsDashboard>();
```

Comment:
@Shazwazza I guess it only need to add `RedirectUrlDashboard` once ;)

Reply:
Thanks :)